### ISSUE TITLE
encoded-compare false positive

### ISSUE THREAD

**Issue by caibirdme** [Antonboom/testifylint#276]
```
xxx_test.go:80:2: encoded-compare: use assert.JSONEq (testifylint)

  assert.Equal(t, contentTypeJson, header.Header.Get("content-Type"))
```

`const contentTypeJson = "application/json"`, this is not a comparison between two json objects

**Comment by ccoVeille** [comment]
This reminds me #248 I worked on.

- #248

**Comment by caibirdme** [comment]
Looking forward for this pr to be merged
